crates/editor/src/element.rs

```diff
@@ -2032,8 +2070,12 @@ impl EditorElement {
                     ||
                     // Selections
                     (is_singleton && scrollbar_settings.selections && editor.has_background_highlights::<BufferSearchHighlights>())
+                    ||
+                    // Symbols Selections
+                    (is_singleton && scrollbar_settings.symbols_selections && editor.has_background_highlights::<BufferSearchHighlights>())
```

Comment:
I believe this should be `DocumentHighlightRead` or `DocumentHighlightWrite`, as shown in `Editor::refresh_document_highlights`